Content.Server/Bed/BedSystem.cs

```diff
@@ -54,6 +55,8 @@ public override void Update(float frameTime)
                 bedComponent.Accumulator -= bedComponent.HealTime;
                 foreach (EntityUid healedEntity in strapComponent.BuckledEntities)
                 {
+                    if (TryComp<MobStateComponent>(healedEntity, out var state) && state.IsDead())
+                        continue;
                     _damageableSystem.TryChangeDamage(healedEntity, bedComponent.Damage, true);
                 }
             }
```

Comment:
I'm looking at the code above this and wondering what is going on:
1. It still heals paused entities?
2. It does 2 separate GetComponents which is really bad

Do you mind fixing those as well?